tmt/utils.py

```diff
@@ -800,17 +800,17 @@ def duration_to_seconds(duration):
 
 def verdict(decision, comment=None, good='pass', bad='fail', problem='warn'):
     """
-    Return verdict in green or red based on the decision
+    Return verdict in green, red or yellow based on the decision
 
-    0 or False ... good (green)
-    1 or True .... bad (red)
+    0 or True .... good (green)
+    1 or False ... bad (red)
     otherwise .... problem (yellow)
     """
 
-    if decision == 0:
-        text = style(bad, fg='red')
-    elif decision == 1:
+    if decision is True or decision is 0:
```

Comment:
seems like some misunderstanding :) When we talked about it last time, you said to me `0` should be true, as it is Unix like :) Anyway, I can change back to 1, not an issue at all.

Reply:
I think the function is already a bit confusing anyway, as it is trying to mix 2 boolean states (True/False) with 3 states represented by integer values (0/1/2). Maybe using an `Enum` with some reasonable values like `good/bad/warn` instead of 0, 1, 2, would be less confusing ...

Happy to discuss it more

Reply:
Yes, the unix-like approach is right. We need `pass`, `fail` and `warn`, so boolean is not enough. What about just removing the `True` and `False` examples to get rid of the confusion?

Reply:
ack, makes sense